src/main/java/com/st/utopia/cancellationservice/controller/CancellationServiceController.java

```diff
@@ -0,0 +1,51 @@
+package com.st.utopia.cancellationservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+import com.st.utopia.cancellationservice.service.TicketService;
+
+import javassist.NotFoundException;
+
+

```

Comment:
The extra whitespace here, and lack of whitespace between arguments in a function below, show that the code has not been even run through the IDE's automatic formatting feature. Always do that before committing.